src/main/java/uk/gov/hmcts/dts/fact/model/deprecated/CourtWithDistance.java

```diff
@@ -42,6 +42,8 @@
     @JsonProperty("magistrate_code")
     private Integer magistratesLocationCode;
     private String slug;
+    @JsonProperty("areas_of_law_spoe")
```

Comment:
At the top of this class, there is a JsonPropertyOrder annotation. It might be worth adding to that too so this field appear after the area of law field in the response